changelog.md

```diff
@@ -12,6 +12,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Template of models (container and component)
 - Icons size to fit into the new models
 
+### Fixed
+
+- Parsing error when have component with object inside object
```

Comment:
when having

a component OR componentS

Reply:
done

Reply:
you forgot to correct the first part.